safe/impact_function_v4/test/test_impact_function.py

```diff
@@ -383,8 +396,6 @@ def test_scenario_directory(self):
             scenario, expected_steps, expected_outputs = read_json_flow(
                 json_file)
             if scenario.get('enable', True):
-                print "Test JSON scenario : "
-                print json_file
```

Comment:
Sorry, but these two prints are useful.
* They are in a test, not in production environment.
* There is no other way to know  which scenario is running as the test will loop over every scenario. It's difficult to debug when we don't know which scenario is failing.

I wonder if the log produces the correct output.